model_downloader/list_topologies.yml

```diff
@@ -1254,7 +1279,10 @@ topologies:
     license: https://github.com/opencv/open_model_zoo/blob/master/LICENSE
 
   - name: "face-person-detection-retail-0002-fp16"
-    description: "Simultaneous detection of Pedestrians and Faces (RMNet with lrelu + SSSSD)."
+    description: >-
+      This is a face and person detector for Retail scenario. It is based onMobileNetV2-like backbone that includes depth-wise convolutions
+      to reduce the amount ofcomputation for the 3x3 convolution block. The two SSD heads from 1/8 (for face detection)and 1/16 (for
```

Comment:
I see there's a space missing before "and". If it's like that in the original text, maybe you could fix that?

Reply:
Actually, I see other similar errors elsewhere, so it's probably something with your conversion script.

Reply:
@IRDonch, lack of space is fixed